terraform/deployments/govuk-test/main.tf

```diff
@@ -5,6 +5,13 @@ terraform {
     region  = "eu-west-1"
     encrypt = true
   }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.13.0"
```

Comment:
We might want to consider using `~>` here, at least until we have an automated solution for keeping these (per-environment) version numbers up to date.

Reply:
Thanks for spotting that, I've fixed that up.